src/pat/modal/tests.js

```diff
@@ -55,6 +57,7 @@ define(["pat-modal"], function(pattern) {
                 expect($modal.find(".header .close-panel").length).toBeTruthy();
                 expect($modal.find(".panel-content p").length).toBe(2);
                 expect($modal.find(".panel-content h4").length).toBe(1);
+                expect($modal.hasClass("modal-active")).toBeTruthy();
             });
 
             it("Modal with a form that has the pat-modal CSS class", function() {
```

Comment:
@cornae yes, I see now I misread your email request, thought you meant a class on the body of the modal vs. a class on the "body" tag.